modules/social_features/social_group/social_group.info.yml

```diff
@@ -21,7 +21,6 @@ dependencies:
   - social:social_event
   - social:social_profile
   - social:social_topic
-  - social:social_group_gvbo
```

Comment:
I can see why it is removed here, but I do think we want this module enabled by default right @ronaldtebrake?
Is that still taken care of @Kingdutch?

Reply:
Yes we want this enabled by default. Install profile should be fine!